module/CMakeLists.txt

```diff
@@ -268,12 +268,16 @@ if(DEAL_II_WITH_CXX20_MODULE)
       set(build_camelcase "Release")
     endif()
 
+    #
     # Define the library. Compile it with the usual flags we also use for
     # all other targets, but do set the DEAL_II_BUILDING_CXX20_MODULE
     # preprocessor variable for a small number of cases where we need to
     # work around things that work differently between module and
     # non-module builds.
-    add_library(${DEAL_II_TARGET_NAME}_module_${build_lowercase} SHARED)
+    #
+
+    add_library(${DEAL_II_TARGET_NAME}_module_${build_lowercase})
```

Comment:
I have removed the `SHARED` keyword consistent with what we do for the regular library.